specification/app/resource-manager/Microsoft.App/preview/2023-04-01-preview/Jobs.json

```diff
@@ -935,6 +938,61 @@
       "format": "int32",
       "type": "integer"
     },
+    "PollingInterval": {
+      "description": "Interval to check each event source in seconds. Defaults to 30s",
+      "format": "int32",
+      "type": "integer"
+    },
```

Comment:
We want to eventually expose polling interval to regular container apps too. I think it'd make sense for it to be a property in the `Scale` object. Since that's not designed yet, maybe we don't support customizing polling interval in jobs yet?

https://github.com/microsoft/azure-container-apps/issues/573#issuecomment-1399024916

Reply:
Thanks